app/src/main/java/com/github/sdp/ratemyepfl/model/review/ReviewOpinion.kt

```diff
@@ -0,0 +1,7 @@
+package com.github.sdp.ratemyepfl.model.review
+
+enum class ReviewOpinion {
```

Comment:
What is this file used for ?

Reply:
It was used during my attempt to keep track for each user of its opinion of each review. I can remove it this poses an issue.

Reply:
If you are not going to use it anymore, you should delete it.

Reply:
Okay, done!